exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<p>consider variable names that include the target name and wavelength rather than 1-4.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>